UserDetails/urls.py

```diff
@@ -15,10 +17,16 @@
         path('members/edit/', MembersEditView.as_view(), name='association_members_edit'),
     ])),
 
-    # Override allauth sign up page with our registration page
-    path('signup/', RegisterView.as_view(), name='account_signup'),
+    path('statistics/', include([
+        path('dining/', DiningHistoryView.as_view(), name='history_lists'),
+        path('dining/<int:page>/', DiningHistoryView.as_view(), name='history_lists'),
+    ])),
+
 
-    path('history/dining/', DiningHistoryView.as_view(), name='history_lists'),
-    path('history/dining/<int:page>/', DiningHistoryView.as_view(), name='history_lists'),
-    path('profile/', ProfileView.as_view(), name='account_profile'),
+    path('settings/account', Settings_Profile_View.as_view(), name='settings_account'),
+    path('settings/', include('allauth.urls')),
+
+    # Override allauth login and sign up page with our registration page
+    path('login/', LoginView.as_view(), name='login_view'),
```

Comment:
Please use `account_login` as name for the login view as that is the same name that allauth uses.
```suggestion
    path('login/', LoginView.as_view(), name='account_login'),
```